src/test/java/uk/gov/hmcts/reform/jobscheduler/jobs/HttpCallJobTest.java

```diff
@@ -49,21 +53,27 @@ public void execute_should_send_http_request_specified_in_job_details() {
         given(context.getJobDetail())
             .willReturn(newJob(HttpCallJob.class).withIdentity("id", "group").build());
 
+        Map<String, String> headers = new HashMap<>();
+        headers.put("X-Custom-Header", "anything");
+        headers.put("ServiceAuthorization", "some-token");
+
         given(actionExtractor.extract(context))
             .willReturn(new HttpAction(
                 "http://localhost:8080/hello-world",
                 HttpMethod.POST,
-                ImmutableMap.of("ServiceAuthorization", "some-token"),
+                headers,
```

Comment:
this can still be an immutable map with 2 headers

Reply:
`headers` as a variable is ok, though.

Reply:
done